### PR TITLE
New import command for Piwigo

### PR DESCRIPTION
**Pull request by samwilson**
This is a new command for importing images from Piwigo into Flickr.

It creates a new photoset into which to put the imported photos, and takes into account Piwigo tags and categories (photosets).

It adds a machine tag for the MD5 checksum of the file being uploaded. This will make it possible in the future to extend this (and other) commands to check for the existence of a file on Flickr before uploading it.

### CODE REVIEW

**Comment by samwilson** [comment]
(Sorry, bug in this; don't review yet!)

**Comment by samwilson** [comment]
Okay, working now. What do you think? (I realise this is probably a fairly un-needed script, but it might be of use to someone out there.)

**Comment by TheFox** [comment]
Thank you for your contribution. I created a [develop](https://github.com/TheFox/flickr-cli/commits/develop) for the newst development version. Please use this branch to fork new features. I switched to PHPStorm IDE, since then I transformed all my PHP projects to PSR-0 standard. Please also apply this standard to your contribution. Then it's ready for merging. :)

**Comment by samwilson** [comment]
Sure no worries! I've rebased on `develop`. Does phpstan support checking for coding standard compliance? Or is it worth adding an extra Travis check with phpcs?

**Comment by TheFox** [comment]
Ok cool. I would use PHPCS for that. PHPStan is a static analyzer.

**Comment by samwilson** [comment]
I've rebased with the new PSR2 formatting.

File: src/TheFox/FlickrCli/Command/PiwigoCommand.php [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+        $this->addOption('piwigo-uploads', null, InputOption::VALUE_REQUIRED, "Path to the Piwigo 'uploads' directory");
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        // Piwigo.
+        $dbConfig = new Configuration();
+        $connectionParams = array(
+            'dbname' => $input->getOption('dbname'),
```
Comment (TheFox) [inline src/TheFox/FlickrCli/Command/PiwigoCommand.php]: Maybe you can move the Piwigo settings to `config.yml`, so we don't have separate options here.

Like this:

```yaml
flickr:
...
piwigo:
  dbname: ...
  user: ...
  password: ...
  host: ...
  driver: pdo_mysql
```

Comment (samwilson) [reply src/TheFox/FlickrCli/Command/PiwigoCommand.php]: Good idea. Done.

**Comment by samwilson** [comment]
This is ready for review, if you've got a mo'. :-)

**Comment by TheFox** [comment]
@samwilson Nice! Sorry for not responding.

**Comment by samwilson** [comment]
No that's ok! :-)

I'm going to get the other common-command stuff done, and then maybe suggest a new release. Will let you know.

**Comment by TheFox** [comment]
Fine :)

**Pull request merged**